ac1/_sources/sql/how_to_run_sql.rst

```diff
@@ -0,0 +1,36 @@
+.. Copyright (C)  Google, Runestone Interactive LLC
+   This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+   International License. To view a copy of this license, visit
+   http://creativecommons.org/licenses/by-sa/4.0/.
+
+
+How to Run SQL Outside of This Textbook
+=======================================
+
+In this section, you learned some basic SQL commands to analyze large datasets.
+In the interactive version of this textbook, there was a SQL interpreter.
+However, when using SQL outside of the textbook, including in your project, you
+will need to run SQL somewhere else. In order to do this, there are many
+options. If you are using SQL for just this course, you can use an online
+interpreter, such as this one.
+
+- `SQL Online Interpreter`_
```

Comment:
Does this allow you to upload datasets?

Reply:
yep! I tested it out with the bikeshare.db and ran a few SQL queries from the textbook. I added a sentence to the instructions to mention that you need to select file -> open db. Feel free to delete it if you think it's unnecessary.

Reply:
I know that the bikeshare data is  inside ac1 > _static, maybe explicitly mention / link it here.